sync/downstream.py

```diff
@@ -159,30 +159,22 @@ def next_action(self):
         latest_try_push = self.latest_valid_try_push
 
         if not latest_try_push:
-            logger.debug("Sync for PR %s has no valid try push" % self.pr)
-            return DownstreamAction.try_push
+            if self.requires_stability_try:
+                logger.debug("Sync for PR %s requires a stability try push" % self.pr)
+                pr = env.gh_wpt.get_pull(self.pr)
+                if pr.merged:  # Wait till PR is merged to push stability run
+                    return DownstreamAction.try_push_stability
+                return DownstreamAction.wait_approved
+            else:
+                return DownstreamAction.try_push
```

Comment:
This I am a bit unsure what to do here. I left it as a `try_push` state because this is now obsolete anyway and won't do anything. Would the ready state be more appropriate? Since if it does not require stability then there is nothing else to be done

Reply:
No, I think only doing a simple try push where there are no test changes but are infra changes still makes sense.

Reply:
Yes that is true, will reimplement the normal try push for this situation